.github/workflows/auto-pr-creation.yml

```
@@ -58,29 +58,46 @@ jobs:
 
           def find_issue_for_task(task_number):
               """Find the GitHub issue for a specific task number"""
-              cmd = ['gh', 'issue', 'list', '--search', f'Task {task_number}:', '--json', 'number,title,milestone']
+              cmd = ['gh', 'issue', 'list', '--search', f'Task {task_number}:', '--state', 'open']
               
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)